### ISSUE TITLE
When I deploy maven project, I get the error Return code is: 413, ReasonPhrase: Request Entity Too Large

### ISSUE THREAD

**Issue by fangzi0813** [travelaudience/kubernetes-nexus#50]
When I deploy maven project to nexus, I get the error:

 Return code is: 413, ReasonPhrase: Request Entity Too Large.

How should I do?

**Comment by GMafra** [comment]
Where you able to deploy it using nexus-proxy?

**Comment by fangzi0813** [comment]
> Were you able to deploy it using nexus-proxy?

yeah, I just use the [https://github.com/travelaudience/nexus-proxy](url), but I don't know how to set some attributes in nexus-proxy like "client_max_body_size" in nginx

**Comment by GMafra** [comment]
I'm having problems setting it with GCP IAM/IAP, maybe you could share some hints? Thanks!

**Comment by fangzi0813** [comment]
Sorry, I have no idea, I disable GCP IAM authentication currently!
